tests/test_wave_01.py

```diff
@@ -158,13 +158,18 @@ def test_moves_movie_from_watchlist_to_empty_watched():
     # Assert
     assert len(updated_data["watchlist"]) == 0
     assert len(updated_data["watched"]) == 1
-    
-    raise Exception("Test needs to be completed.")
+
+    # Assert added
+    assert updated_data ["watched"]==[{
```

Comment:
```suggestion
    assert updated_data["watched"] == [{
```

Incorrect usage of whitespaces. It's a small thing, but as you write more code (and eventually contribute to a large codebase) it's important to be consistent with whitespaces so that the whole project stays neat.

Please review the [PEP8 Style Guide](https://peps.python.org/pep-0008/#whitespace-in-expressions-and-statements) to see how we should use whitespaces in Python.